main.py

```
@@ -193,10 +193,21 @@ async def handler(data):
         payload = data.get("data")
         peer_addr = data.get("_peer_addr", "unknown")
 
-        if payload is None and msg_type in ("hello", "chain_request", "chain_response"):
+        if payload is None and msg_type in ("chain_request", "chain_response"):
             return
```

Ruff (0.16.1)

[warning] Do not implicitly `return None` in function able to return non-`None` value

Add explicit `None` return value

(RET502)